### DISCUSSION TITLE
can you tell me how to download this mod on 1.18.2?

### DISCUSSION THREAD

**Discussion by ghost** [MartinTheDragon/Nuclear-Tech-Mod-Remake#65] [Q&A] [answered]
this is my first time on the guide hub and I don’t know where to click here to find HBM on 1.18.2 and download

**Comment by MartinTheDragon** [comment, marked as answer]
Go to the [releases page](https://github.com/MartinTheDragon/Nuclear-Tech-Mod-Remake/releases), for each update, scroll down on the uppermost panel until you see "Assets", click on the arrow to expand the list, then download the jar file.
